openmc/settings.py

```diff
@@ -139,6 +139,8 @@ class Settings(object):
         The elastic scattering model to use for resonant isotopes
     volume_calculations : VolumeCalculation or iterable of VolumeCalculation
         Stochastic volume calculation specifications
+    no_nu : bool
```

Comment:
I find the name "no_nu" to not be very descriptive. I'd prefer something a little more verbose like "create_fission_neutrons" that is True by default.